### DISCUSSION TITLE
Awesome Project!

### DISCUSSION THREAD

**Discussion by lodoreds** [HelgeGehring/femwell#61] [General]
Hi guys,
What an amazing solver! I tried it a little and I already love its speed and accuracy.
I just wanted to thank you for sharing this project and let you know that I would be glad to contribute with a few ideas and suggestions.
Cheers,
Lodovico

**Comment by HelgeGehring** [comment]
Thanks a lot! :partying_face: 
We're happy about every contribution :)
